src/main.js

```diff
@@ -169,21 +170,32 @@ class MiniGraphCard extends LitElement {
   }
 
   render({ config } = this) {
+    const aspectRatio = {};
+    if (config.aspect_ratio) {
+      aspectRatio['--aspect-ratio'] = config.aspect_ratio;
+    } else {
+      aspectRatio.display = 'inline';
+    }
+
     return html`
-      <ha-card
-        class="flex"
-        ?group=${config.group}
-        ?fill=${config.show.graph && config.show.fill}
-        ?points=${config.show.points === 'hover'}
-        ?labels=${config.show.labels === 'hover'}
-        ?labels-secondary=${config.show.labels_secondary === 'hover'}
-        ?gradient=${config.color_thresholds.length > 0}
-        ?hover=${config.tap_action.action !== 'none'}
-        style="font-size: ${config.font_size}px;"
-        @click=${e => this.handlePopup(e, config.tap_action.entity || this.entity[0])}
-      >
-        ${this.renderHeader()} ${this.renderStates()} ${this.renderGraph()} ${this.renderInfo()}
-      </ha-card>
+      <div 
+        id="aspect-ratio"
+        style="${styleMap(aspectRatio)}">
```

Comment:
TBH I'm not convinced that this is worth to add a new dependency

Reply:
You're absolutely right about that, good catch.  I removed that dependency.